### ISSUE TITLE
Basic template fails at `\faculty` control sequence when also calling `\maketitle` (was caused by missing `csquotes` package)

### ISSUE THREAD

**Issue by Noxdor** [tud-cd/tudscr#99]
I am working with the tudscr package and template for the first time and unfortunately I can not get it to work. I reduced the template now as far as I could and my current and hopefully only issue is that the `\faculty` control sequence is undefined. This error only appears when I call the `\maketitle` control sequence. This is my setup:

```latex
\documentclass{tudscrreprt}


\begin{document}
%% Daten zur Titelseite
\thesis{Diplomarbeit}
\graduation[Dipl."~Ing.]{Diplomingenieur}
\author{Dagobert Duck}
\placeofbirth{Entenhausen}
\dateofbirth{01.01.1900}
\faculty{Fakultät Verkehrswissenschaften \enquote{Friedrich List}}
\institute{Institut für Automobiltechnik Dresden}
\chair{Professur für Fahrzeugmechatronik}
\matriculationnumber{1234}
\title{Strukturelle Optimierung der Betriebsstrategie eines Geldspeichers}
\professor{Prof. Dr.-Ing. B. Bäker}
\supervisor{E. Delstein}
\date{\today}
\maketitle % if I comment this out, the run succeeds but obviously nothing is printed since I am assuming this call creates the actual title page.

\end{document}
```

Any idea if I might be missing a package or am using an outdated template?

**Comment by Noxdor** [comment]
When I comment out `\faculty` I am successfully generating a title page.

**Comment by Noxdor** [comment]
Alright, I figured it out. the issue is not `\faculty` but `\enquote` which depends on the `csquotes` package. Adding this package with `\usepackage` resolves the error. I was thrown off because the original template I used was a lot bigger than what I posted and something else was causing an error, which I understand now was the following code:
```latex
\ifpdftex{
  \usepackage[ngerman=ngerman-x-latest]{hyphsubst}
}{}
```
Commenting that out and leaving the rest of the template as is leads to a successful compilation.
